site/viamguides/src/monitoring-automation-elastic/monitoring-automation-elastic.md

```diff
@@ -0,0 +1,431 @@
+author: HipsterBrown
+id: monitoring-automation-elastic
+summary: Monitoring and automate the physical world using sensor data from machines with Elasticsearch and Kibana
+categories: Getting-Started, Developer
+environments: web
+status: Published 
+feedback link: https://github.com/viam-labs/viamcodelabs/issues
+tags: Getting Started, Developer, Data
+
+# Use Sensor Data with Elastic, Kibana, and Webhooks
+<!-- ------------------------ -->
+## Overview 
+Duration: 1
+
+Gathering sensor data starts out simple but over time it can be overwhelming to sift through all that information, especially as more and more sensors come online. If you've worked on a comprehensive web service in the past, you might already know about [Elasticsearch](https://www.elastic.co/elasticsearch) and the rest of the [ELK stack](https://www.elastic.co/elastic-stack) for indexing and analyzing data from any source, including sensors connected to Viam machines!
+One neat feature of the ELK stack is configurable [alerting rules](https://www.elastic.co/guide/en/kibana/current/alerting-getting-started.html) and [actions](https://www.elastic.co/guide/en/kibana/current/alerting-getting-started.html#alerting-concepts-actions) that can be used to automate notifications or even affect the physical world using one of Viam's SDKs.
+
+In this codelab, you'll learn how to continually index sensor data from Viam into Elasticsearch and display an alert in the real world. 
```

Comment:
Suggestion to include a high-level technical workflow diagram to show Elastic, Viam, maybe GCP and Zrok. Could also be used in a demo video to abstract some of the setup details (in the video).